### PR TITLE
Isabella

### PR DESCRIPTION
**Pull request by isabella0428**
Hi Youchen,

Please check my commit when u are available! 
Feel free to point out anything that u don't feel appropriate :)
Thx for ur advice in advance.

Best Regards
Yi Lyu

### CODE REVIEW

File: portfolio/src/main/webapp/intro.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                    Programming Language <br>
+                    <br>
+                    Framework
+                </div>
+
+                <div class="middleColumn">
+                    C/C++, Python, Java, JavaScript, Python, HTML, CSS, SQL <br>
+                    <br>
+                    <br>
+                    Tensorflow,Keras, MATLAB, LATEX, FLASK,d3.js
```
Comment (Time1ess) [inline portfolio/src/main/webapp/intro.html:108]: Nit: TensorFlow, LaTex, Flask, D3.js.

You may need to pay attention to those tech terms, though it doesn't affect anything, but it gives a sense about how careful you are :P.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,28 +1,15 @@
...
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+Author: Yi Lyu
+Date:   2020.07.08
+Email:  isabella_aus_china@outlook.com
+*/
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Jump to intro page from start page
```
Comment (Time1ess) [inline portfolio/src/main/webapp/script.js:7]: Nit: // Jumps to ...

Normally we'd like to start function or method comments with third-person verbs.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -1,28 +1,15 @@
...
+function jumpToIntro() {
+    window.location.href="intro.html";
 }
+
+// Jump to main page
```
Comment (Time1ess) [inline portfolio/src/main/webapp/script.js:12]: Ditto

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -1,28 +1,15 @@
...
+Date:   2020.07.08
+Email:  isabella_aus_china@outlook.com
+*/
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Jump to intro page from start page
+function jumpToIntro() {
+    window.location.href="intro.html";
```
Comment (Time1ess) [inline portfolio/src/main/webapp/script.js:9]: Nit: Maybe make it an absolute path? Like /intro.html.

Not sure about this, but it looks to me that this is a relative URL navigation, though you don't have hierarchical structure for now, you may encounter them later. Just to make sure that you know the difference between relative URL navigation and absolute URL navigation.

**Comment by isabella0428** [comment]
I will modify my code according to ur suggestion in the next commit :D
Thx for ur advice Youchen.

**Pull request merged**